src/BjyAuthorize/Provider/Role/ObjectRepositoryProvider.php

```diff
@@ -10,23 +10,23 @@
 
 use BjyAuthorize\Acl\HierarchicalRoleInterface;
 use BjyAuthorize\Acl\Role;
-use Doctrine\Common\Persistence\ObjectRepository;
+use Doctrine\Persistence\ObjectRepository;
```

Comment:
so we need also a new dev dependency
`doctrine/persistence: ^1.3.3 || ^2.0`

Reply:
You are right, I replaced the dependency to be more specific.

Reply:
i think the composer.lock, need the update too

Reply:
Oh, I actually deleted it, as I thought vthat the commit was accidental, as for libraries it's usually of no effect as per https://getcomposer.org/doc/02-libraries.md#lock-file

I can restore and update it.